study.md

```diff
@@ -22,7 +22,9 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, define ORM and explain why using an ORM is valuable.
 
 ```md
-<!-- your response here -->
+An object relational mapper is a library for object oriented programming, which aids in translating between the languages memory objects and relational databases.
```

Comment:
It's not a library but ORM libraries exist.  Object relational-mapping (ORM) is a method of representing arbitrary data as objects in a programming language.